src/python/pants/notes/2.10.x.md

```diff
@@ -1,5 +1,75 @@
 # 2.10.x Release Series
 
+## 2.10.0rc0 (Feb 07, 2022)
+
+### New Features
+
+* Add `--fmt-only`, `--check-only`, and `--lint-only` ([#14356](https://github.com/pantsbuild/pants/pull/14356))
+
+* Add support for building PyOxidizer apps ([#14183](https://github.com/pantsbuild/pants/pull/14183))
+
+* Adds `run` support for `deploy_jar` targets ([#14352](https://github.com/pantsbuild/pants/pull/14352))
+
+* New `[docker].executable_search_paths` option. ([#14360](https://github.com/pantsbuild/pants/pull/14360))
+
+* Support Go 1.18 fuzz test targets ([#14357](https://github.com/pantsbuild/pants/pull/14357))
+
+* Validate Python user lockfiles & improve tool lockfile error message ([#14353](https://github.com/pantsbuild/pants/pull/14353))
+
+* Allow using Yapf with `update-build-files` rather than Black ([#14378](https://github.com/pantsbuild/pants/pull/14378))
+
+### User API Changes
+
+* Graduate Docker backend from experimental status. ([#14310](https://github.com/pantsbuild/pants/pull/14310))
+
+* Graduate the Dockerfile linter Hadolint from experimental status. ([#14311](https://github.com/pantsbuild/pants/pull/14311))
+
+* Graduate Thrift Python support to be stable with `pants.backend.codegen.thrift.apache.python` ([#14389](https://github.com/pantsbuild/pants/pull/14389))
```

Comment:
Should the Thrift PRs be in "New Features" since Thrift is new and not really a change to an existing API?

Reply:
Yeah true